src/lib/input/input.ts

```diff
@@ -52,10 +52,10 @@ let nextUniqueId = 0;
 @Directive({
   selector: `input[mdInput], textarea[mdInput], input[matInput], textarea[matInput]`,
   host: {
-    'class': 'mat-input-element',
+    'class': 'mat-input-element mat-form-field-autofill-float',
     // Native input properties that are overwritten by Angular inputs need to be synced with
     // the native input element. Otherwise property bindings for those don't work.
-    '[id]': 'id',
+    '[attr.id]': 'id',
```

Comment:
Becuase `id` is also an input
https://github.com/mmalerba/material2/blob/32df1d401bd97ce0d2a5fcf9a5e1c2bac7ddf076/src/lib/input/input.ts#L105